tests/network/bgp/evpn/libevpn.py

```
@@ -17,3 +79,141 @@ def cudn_evpn_subnets() -> list[str]:
+    with contextlib.ExitStack() as stack:
+        active_conns = []
+        for server_ip in server_ips:
+            active_conns.append(
+                stack.enter_context(
+                    cm=_evpn_workloads_connection(
+                        endpoint=endpoint,
+                        vm=vm,
+                        server_ip=str(server_ip),
+                    ),
+                )
+            )
```

Ruff (0.15.12)

[warning] Use a list comprehension to create a transformed list

(PERF401)